static/js/components/notifications/FooterNotifications.js

```diff
@@ -0,0 +1,10 @@
+// @flow
+import React from "react"
+import NotificationContainer from "./NotificationContainer"
+import { ALERT_TYPE_ERROR, ALERT_TYPE_SUCCESS } from "../../constants"
+
+const FooterNotifications = () => (
```

Comment:
Is this component necessary? Could we just render `<NotificationContainer ... />` directly in root.js?